packages/dev/docs/pages/react-spectrum/versioning.mdx

```diff
@@ -0,0 +1,124 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+---
+category: Introduction
```

Comment:
Do we want this in the introduction category or concepts (along with styling, layout, theming, etc.). Felt it didn't really fit with the other concepts but I could be convinced.

Reply:
yeah but architecture isn't under RSP, it's at the root of the site.

Reply:
+1 to introduction